README.md

```diff
@@ -49,6 +49,27 @@ Building from command line
     $ mvn clean verify
 ````
 
+Running from the command line
+------------------------------
+1. Choose a connection type from "Managing connection types" section below, and then set those environment variables in your terminal prior to continuing
+
+2. Make sure to build the server using the steps above in the "Building from command line" section
+
+3. `cd` into the build directory of the project: `/org.eclipse.jdt.ls.product/target/repository`
+
+4. Prior to starting the server, make sure that your socket (TCP or sock file) server is running for both the IN and OUT sockets. You will get an error if the JDT server cannot connect on your ports/files specified in the environment variables
+
+5. To start the server in the active terminal, run:
+```bash
+java -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=1044 -Declipse.application=org.eclipse.jdt.ls.core.id1 -Dosgi.bundles.defaultStartLevel=4 -Declipse.product=org.eclipse.jdt.ls.core.product -Dlog.protocol=true -Dlog.level=ALL -noverify -Xmx1G -jar ./plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar -configuration ./config_linux -data /path/to/data
```

Comment:
The version of launcher (_org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar_) may change as the eclipse versions changes perhaps we should have a note of it.